### DISCUSSION TITLE
Why are the versions I compiled are all 1.0.0, even with Tag 1.21.1

### DISCUSSION THREAD

**Discussion by ShiningSong1** [ipxe/ipxe#1022] [Q&A]
Why are the versions I compiled are all 1.0.0, even with Tag 1.21.1？

**Comment by NiKiZe** [comment]
You have not provided any information on what you have done or what the output is.

But likely duplicate of what is described and answered in https://github.com/ipxe/ipxe/discussions/454#discussioncomment-1296930
